src/test/java/EmployeeTest.java

```diff
@@ -0,0 +1,151 @@
+    public void checkGetEmployeesByIds(int id, String employeeName, int employeeSalary, int employeeAge, String profileImage) {
+        Response employeeByIdResponse = getEmployeeByIdResponse(id);
+
+        assertThat(employeeByIdResponse.statusCode()).as("check response status code").isEqualTo(200);
+
+        Employee employee = employeeByIdResponse.as(Employee.class);
+
+        SoftAssertions softly = new SoftAssertions();
+            softly.assertThat(employee.getData().getEmployee_name()).isEqualTo(employeeName);
+            softly.assertThat(employee.getData().getEmployee_salary()).isEqualTo(employeeSalary);
+            softly.assertThat(employee.getData().getEmployee_age()).isEqualTo(employeeAge);
+            softly.assertThat(employee.getData().getProfile_image()).isEqualTo(profileImage);
+        softly.assertAll();
+    }
+
+    @Test
+    public void checkForEmployeeWithNotExistingId() {
+        int notExistingId = 25;
+        Response employeeByIdResponse = getEmployeeByIdResponse(notExistingId);
+
+        assertThat(employeeByIdResponse.statusCode()).as("check response status code").isEqualTo(200);
+
+        Employee employee = employeeByIdResponse.as(Employee.class);
+
+        SoftAssertions softly = new SoftAssertions();
+            softly.assertThat(employee.getData()).isNull();
+            softly.assertThat(employee.getStatus()).isEqualTo("failure");
+            softly.assertThat(employee.getMessage()).isEqualTo("Failure! There is no such employee.");
+        softly.assertAll();
+    }
+
+    @Test
+    public void checkForNegativeAgeValueOfEmployees() {
+        Response allEmployeesResponse = getAllEmployeesResponse();
+
+        assertThat(allEmployeesResponse.statusCode()).as("check response status code").isEqualTo(200);
+
+        Employees employees = allEmployeesResponse.as(Employees.class);
+
+        assertThat(employees.getData().stream().filter(it -> it.getEmployee_age() < 0)).isEmpty();
+    }
+
+    @Test
+    public void checkForNegativeSalaryValueOfEmployees() {
+        Response allEmployeesResponse = getAllEmployeesResponse();
+
+        assertThat(allEmployeesResponse.statusCode()).as("check response status code").isEqualTo(200);
+
+        Employees employees = allEmployeesResponse.as(Employees.class);
+
+        assertThat(employees.getData().stream().filter(it -> it.getEmployee_salary() < 0)).isEmpty();
+    }
+
+    @Test
+    public void checkForDigitsInEmployeesNames() {
+        Response allEmployeesResponse = getAllEmployeesResponse();
+
+        assertThat(allEmployeesResponse.statusCode()).as("check response status code").isEqualTo(200);
+
+        Employees employees = allEmployeesResponse.as(Employees.class);
+
+        assertThat(employees.getData().stream().filter(it -> it.getEmployee_name().matches(".*\\d.*"))).isEmpty();
+    }
+
+    @Test
+    public void testCreateNewEmployee() {
+        JSONObject newEmployeeData = new JSONObject().
```

Comment:
Please use a dedicated model for this case, EmployeeData should work. I would suggest to implement builder pattern in this class. Lombok allows to create default builders easily. After that you can easily build desired test object and then serialise it with `gson.toJson(obj)`